phpcs-ruleset.xml

```diff
@@ -78,6 +76,7 @@
     <!-- Views handlers not strictly follow Drupal function name conventions. -->
     <rule ref="Drupal.NamingConventions.ValidFunctionName">
         <exclude-pattern>profiles/common/modules/custom/ecas/ecas_extra/includes/views/handlers/*.inc</exclude-pattern>
+        <exclude-pattern>profiles/common/modules/features/multisite_twitterblock/plugins/*.inc</exclude-pattern>
```

Comment:
This is not a Views handler, but a Context reaction. It's fine to update the documentation to say that both Views handlers and Context reactions do not follow the conventions.

There is only one file affected, so don't use a wildcard here. The plugins/ folder is generic. If more plugins are added in the future we want to be sure they are scanned for this coding standard.